app/src/sharedTest/java/org/oppia/app/settings/profile/ProfileRenameActivityTest.kt

```diff
@@ -75,6 +79,24 @@ class ProfileRenameActivityTest {
     }
   }
 
+  @Test
+  fun testProfileRenameActivity_inputNewName_configurationChange_checkSaveIsEnabled() {
+    ActivityScenario.launch<ProfileRenameActivity>(ProfileRenameActivity.createProfileRenameActivity(context, 1)).use {
+      onView(allOf(withId(R.id.input), isDescendantOfA(withId(R.id.input_name)))).perform(typeText("James"))
+      onView(isRoot()).perform(orientationLandscape())
+      onView(withId(R.id.profile_rename_save_button)).check(matches(isEnabled()))
+    }
+  }
+
+  @Test
+  fun testProfileRenameActivity_inputNewName_configurationChange_inputTextExists() {
```

Comment:
@aggarwalpulkit596 How are you testing this test cases? Because one of these test case is still failing and actually it has been written incorrectly and therefore it should fail, so I am not able to understand why are you assigning these failed test cases. I only conclusion that I was able make was that you are writing test cases and pushing it and checking if circle CI passes them or not.

Before making a push always run your test cases on your local machine first. In this case if they are passing please share the screenshot here.

Reply:
Yes they are running fine on my local machine will share a screenshot of them right away

Reply:
@rt4914 and also these test passed on Travis CI

Reply:
@rt4914 apologies I tried something locally and pushed that one by mistake and didn't check because Circle-CI test was passed

Reply:
> @rt4914 apologies I tried something locally and pushed that one by mistake and didn't check because Circle-CI test was passed

I am not able to understand. If the tests are passing your local machine, please share the screenshot so that we can be on the same page.

Reply:
Okay, after this change test cases are passing. Please make sure that your test cases are always correct as it wastes a lot of time.